docs/content/guides/dagster/automated_pipelines.mdx

```diff
@@ -0,0 +1,162 @@
+
+We also have an asset for the expenses, which also depends on the transactions asset. This asset has a freshness policy of 120 minutes, or two hours at any time, since it did not specify a schedule. The expenses asset will require the transactions data to be refreshed in order to comply with the freshness policy.
+
+```python file=/guides/dagster/automating_pipelines/declare_schedule.py
+from dagster import (
+    AssetSelection,
+    Definitions,
+    FreshnessPolicy,
+    asset,
+    build_asset_reconciliation_sensor,
+)
+
+@asset
+def transactions():
+    pass
+
+@asset(freshness_policy=FreshnessPolicy(maximum_lag_minutes=60, cron_schedule="0 9 * * *"))
+def sales(transactions):
+    pass
+
+@asset(freshness_policy=FreshnessPolicy(maximum_lag_minutes=120))
+def expenses(transactions):
+    pass
+
+update_sensor = build_asset_reconciliation_sensor(
+    name="update_sensor", asset_selection=AssetSelection.all()
+)
+
+defs = Definitions(assets=[transactions, sales, expenses], sensors=[update_sensor])
+```
+
+---
+
+## My data should be updated every time something happens
+
+### I want my data to be updated when an event happens
+
+Let’s say you have sales pipeline data that you want updated every time a customer submits an RFP on your website. Dagster offers [sensors](/concepts/partitions-schedules-sensors/sensors) that allow you to kick off jobs based on some external change, such as a new file in an S3 bucket, if some other asset is updated, or if a system is down.
+
+```python file=/guides/dagster/automating_pipelines/s3_sensor.py
+from dagster_aws.s3.sensor import get_s3_keys
+from dagster import sensor
+
+@sensor(job=my_job)
+def my_s3_sensor(context):
+    since_key = context.cursor or None
+    new_s3_keys = get_s3_keys("my_s3_rfp_bucket", since_key=since_key)
+    if not new_s3_keys:
+        return SkipReason("No new s3 files found for bucket my_s3_rfp_bucket.")
+    last_key = new_s3_keys[-1]
+    run_requests = [RunRequest(run_key=s3_key, run_config={}) for s3_key in new_s3_keys]
+    context.update_cursor(last_key)
+    return run_requests
+```
+
+### I want some assets to materialize every time other assets materialize
+
+If every time you materialize Airbyte assets you want to execute a dbt asset, [asset sensors](/concepts/partitions-schedules-sensors/asset-sensors#asset-sensors-1) can be used to automate that process. Asset sensors can be used across [code locations](/concepts/code-locations#code-locations), so if you are managing some Airbyte assets in one code location and DBT assets in another, you can use asset sensors to trigger jobs to run.
+
+Dagster provides [reconciliation sensors](/concepts/partitions-schedules-sensors/asset-sensors#asset-reconciliation-sensors) that automatically materialize assets if they are ‘unreconciled’. One of the ways assets are considered unreconciled if any of the asset’s upstream dependencies have been materialized more recently than it has.
```

Comment:
🤔 I didn't catch this in Notion - sorry! But perhaps it's worth mentioning that a reconciliation sensor is a type of asset sensor here. I had to re-read this and hover over the link to see it also goes to the asset sensors page, as I thought I'd missed something.